release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -170,6 +170,31 @@ The {product-version} tested maximum for the number of Pods per node is 500.
 Use the link:https://access.redhat.com/labs/ocplimitscalculator/[{product-title}
 Limit Calculator] to estimate cluster limits for your environment.
 
+[id="ocp-4-4-developer-experience"]
+=== Developer experience
+
+[id="ocp-4-4-automatic-image-pruning"]
+==== Automatic image pruning
+
+Image pruning is now performed automatically. This is managed by the Registry
+Operator, which creates a CronJob to run periodic image pruning.
```

Comment:
Late breaking issue - cluster admins will need to enable this feature in 4.4.

See https://bugzilla.redhat.com/show_bug.cgi?id=1825284